pkg/cluster/cluster.go

```diff
@@ -117,17 +117,17 @@ func New(ctx context.Context, log *logrus.Entry, _env env.Interface, db database
 		return nil, err
 	}
 
-	localFPAuthorizer, err := _env.FPAuthorizer(_env.TenantID(), _env.Environment().ResourceManagerEndpoint)
+	localFPAuthorizer, err := _env.FPAuthorizer(_env.TenantID(), _env.Environment().ResourceManagerEndpoint+"/.default")
 	if err != nil {
 		return nil, err
 	}
 
-	fpAuthorizer, err := _env.FPAuthorizer(subscriptionDoc.Subscription.Properties.TenantID, _env.Environment().ResourceManagerEndpoint)
+	fpAuthorizer, err := _env.FPAuthorizer(subscriptionDoc.Subscription.Properties.TenantID, _env.Environment().ResourceManagerEndpoint+"/.default")
```

Comment:
I personally would like to see this string put into a const somewhere and used everywhere so that the significance of the string is documented.

Reply:
I was originally pushing back on doing this since these scope strings are temporary and will go away once we fully migrate to the newer Azure SDK that uses the [TokenCredential interface](https://github.com/Azure/azure-sdk-for-go/blob/sdk/azcore/v1.4.0/sdk/azcore/internal/exported/exported.go#L80) instead of [ADAL-based Authorizers](https://pkg.go.dev/github.com/Azure/go-autorest/autorest#Authorizer).  But since you're the 3rd person to ask, I'll go ahead and add some scope fields to the `AROEnvironment` struct.